src/api.ts

```diff
@@ -23,6 +23,7 @@
     .then(res => res.json());
 }
 
-export const getTodos = () => get<Todo[]>('/todos');
+export const getTodos: () => Promise<Todo[]> = () => get<Todo[]>('/todos');
 
-export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+export const getUser: (userId: number) => Promise<User> = (userId: number) =>
+  get<User>(`/users/${userId}`);
```

Comment:
Ensure that the `/users/{userId}` endpoint returns data in the expected format for `User`. This is important for user-related functionality.